db/migrate/20150731161351_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :username
```

Comment:
You might want to validate uniqueness here so two users can't have the same username